lessons/pt/basics/enum.md

```diff
@@ -1,13 +1,13 @@
 %{
-  version: "1.8.0",
+  version: "1.9.1",
   title: "Enum",
   excerpt: """
   Um conjunto de algoritmos para fazer enumeração em coleções.
   """
 }
 ---
 
-## Enum
+# Overview
```

Comment:
I think we translate `Overview` -> `Visão Geral`:
https://github.com/elixirschool/elixirschool/blob/master/lessons/pt/basics/iex_helpers.md?plain=1#L9

```suggestion
# Visão Geral
```

Makes sense?